DEPS

```diff
@@ -11,7 +11,7 @@ deps = {
   "vendor/python-patch": "https://github.com/svn2github/python-patch@a336a458016ced89aba90dfc3f4c8222ae3b1403",
   "vendor/omaha":  "https://github.com/brave/omaha.git@e4263050ed24e92f4d3ed0b6874538f811011c5b",
   "vendor/sparkle": "https://github.com/brave/Sparkle.git@c0759cce415d7c0feae45005c8a013b1898711f0",
-  "vendor/bat-native-ledger": "https://github.com/brave-intl/bat-native-ledger@fd1df356d6e5a1f105b6743a9a0b29e5abcd1356",
+  "vendor/bat-native-ledger": "https://github.com/brave-intl/bat-native-ledger@ebbc1c56d8d349069f8dcec2edc3433ff87eb42f",
```

Comment:
?

Reply:
This rolls in a compilation fix for brave-intl/bat-native-ledger#189

Reply:
Perfect- this is the hash for 0.57.x on bat-native-ledger (which has the fix on Windows for std::sort error)